src/utils.js

```diff
@@ -474,6 +474,8 @@ export const getIdentifiers = (identifiers = [], type, identifierTypesById) => {
     }
   });
 
+  if (!result.length) return null;
```

Comment:
This change is required, because previous logic wasn't correct. In prev implementation we will never see Not available label, because isbn and issn always returns `true` from `getIdentifiers`.